### ISSUE TITLE
Make it pip / snap installable

### ISSUE THREAD

**Issue by nottrobin** [ubuntudesign/asset-uploader#3]
For ease of use, this tool should probably be easily installable, e.g.:

``` bash
$ pip install assets-cli
# or
$ snap install assets-cli
# ...
$ assets -s example.com -t x1dex21x upload my-image.png
```

**Comment by WillMoggridge** [comment]
Oh, snap would be a fantastic idea if we can get that working

**Comment by nottrobin** [comment]
Resolved by #12, #13 and #14
